pkg/minikube/cluster/cluster_test.go

```diff
@@ -57,8 +57,8 @@ func TestCreateHost(t *testing.T) {
 		t.Fatalf("Machine is not running. State is: %s", s)
 	}
 
-	if h.DriverName != "virtualbox" {
-		t.Fatalf("Wrong driver name: %v. Should be virtualbox.", h.DriverName)
+	if h.DriverName != "virtualbox" || h.DriverName != "vmwarefusion" {
```

Comment:
I'd like if you made an array of supported drivers that could be used when validating the flag and here in the test